run_tests.py.in

```diff
@@ -383,6 +385,7 @@ def run_test(test):
     switches = switches.replace('re2ocaml', 're2c --lang ocaml')
     switches = switches.replace('re2py', 're2c --lang python')
     switches = switches.replace('re2rust', 're2c --lang rust')
+    switches = switches.replace('re2swift, 're2c --lang swift')
```

Comment:
Missing `'` after `'re2swift`.

Reply:
There appears to be more problems than just that missing quote terminator, `tests` reports all the swift tests as missing and I'm not sure why yet.

Reply:
Ah, it's probably because you forgot to add .Swift file extension in the run_tests.py.in. I always get int this trap myself.

Reply:
(see line 302)

Reply:
Okay, the missing issue was because I neglected to add the swift extensions to the glob. Now that that's fixed I'm still getting test failures, still not sure why yet.

Reply:
Ohhhhh it's because diff is noticing the `sourceLocation`s are different because of how I generated the tests.

Reply:
This particular issue should be solved now.